### PR TITLE
Fixed #27392 -- Remove "Tests that", "Ensures that", etc. from test docstings

### PR DESCRIPTION
**Pull request by roman-dmytriv**

### CODE REVIEW

File: tests/admin_changelist/tests.py [hunk 1]
```diff
@@ -64,8 +64,8 @@ def _mocked_authenticated_request(self, url, user):
 
     def test_select_related_preserved(self):
         """
-        Regression test for #10348: ChangeList.get_queryset() shouldn't
-        overwrite a custom select_related provided by ModelAdmin.get_queryset().
+        ChangeList.get_queryset() shouldn't
```
Comment (timgraham) [inline tests/admin_changelist/tests.py]: The line lengths should be reflowed to 79 characters so there isn't a short line followed by a long line.

File: tests/admin_changelist/tests.py [hunk 2]
```diff
@@ -192,12 +192,12 @@ def test_result_list_html(self):
 
     def test_result_list_editable_html(self):
         """
-        Regression tests for #11791: Inclusion tag result_list generates a
+        Inclusion tag result_list generates a
         table and this checks that the items are nested within the table
-        element tags.
-        Also a regression test for #13599, verifies that hidden fields
+        element tags (#11791.
+        Verifies that hidden fields
```
Comment (timgraham) [inline tests/admin_changelist/tests.py]: "Verifies that" is a similar phrase to avoid.

File: tests/auth_tests/test_views.py [hunk 3]
```diff
@@ -672,9 +672,8 @@ def test_login_csrf_rotate(self):
 
     def test_session_key_flushed_on_login(self):
         """
-        To avoid reusing another user's session, ensure a new, empty session is
-        created if the existing session corresponds to a different authenticated
-        user.
+        A new, empty session is created if the existing session corresponds to a different authenticated
```
Comment (timgraham) [inline tests/auth_tests/test_views.py]: Wrap at 79 characters please.

File: tests/custom_managers/tests.py [hunk 4]
```diff
@@ -605,9 +606,10 @@ def test_filtered_default_manager(self):
         self.assertEqual(RestrictedModel.plain_manager.count(), 1)
 
     def test_delete_related_on_filtered_manager(self):
-        """Deleting related objects should also not be distracted by a
-        restricted manager on the related object. This is a regression
-        test for #2698."""
+        """
+        Deleting related objects should also not be distracted by a
+        restricted manager on the related object(#2698).
```
Comment (timgraham) [inline tests/custom_managers/tests.py]: missing space before (

File: tests/defer_regress/tests.py [hunk 5]
```diff
@@ -83,7 +83,7 @@ def test_basic(self):
         self.assertEqual(results[0].child.name, "c1")
         self.assertEqual(results[0].second_child.name, "c2")
 
-        # Regression for #16409 - make sure defer() and only() work with annotate()
+        # Make sure defer() and only() work with annotate() (#16409).
```
Comment (timgraham) [inline tests/defer_regress/tests.py]: "Make sure" is a phrase to avoid

File: tests/defer_regress/tests.py [hunk 6]
```diff
@@ -130,7 +130,7 @@ def test_ticket_23270(self):
             self.assertEqual(1, obj.derived.base_ptr_id)
 
     def test_only_and_defer_usage_on_proxy_models(self):
-        # Regression for #15790 - only() broken for proxy models
+        # only() broken for proxy models (#15790).
```
Comment (timgraham) [inline tests/defer_regress/tests.py]: only() works with proxy models

File: tests/file_uploads/views.py [hunk 7]
```diff
@@ -121,7 +121,7 @@ def file_upload_quota_broken(request):
 
 def file_upload_getlist_count(request):
     """
-    Check the .getlist() function to ensure we receive the correct number of files.
+    Check the .getlist() function to receive the correct number of files.
```
Comment (timgraham) [inline tests/file_uploads/views.py]: This isn't grammatically correct.

File: tests/file_storage/tests.py [hunk 8]
```diff
@@ -882,10 +882,9 @@ def tearDown(self):
         shutil.rmtree(self.storage_dir)
 
     def test_directory_with_dot(self):
-        """Regression test for #9610.
-
+        """
         If the directory name contains a dot and the file name doesn't, make
-        sure we still mangle the file name instead of the directory name.
+        sure we still mangle the file name instead of the directory name ((#9610).
```
Comment (timgraham) [inline tests/file_storage/tests.py]: double ((

File: tests/migrations/test_operations.py [hunk 9]
```diff
@@ -1514,7 +1514,7 @@ def test_remove_index_state_forwards(self):
 
     def test_alter_field_with_index(self):
         """
-        Test AlterField operation with an index to ensure indexes created via
+        AlterField operation with an index to indexes created via
```
Comment (timgraham) [inline tests/migrations/test_operations.py]: This isn't grammatically correct.

File: tests/model_fields/test_uuid.py [hunk 10]
```diff
@@ -153,7 +153,7 @@ def test_underlying_field(self):
         self.assertEqual(related.uuid_fk.pk, related.uuid_fk_id)
 
     def test_update_with_related_model_instance(self):
-        # regression for #24611
+        # (#24611).
```
Comment (timgraham) [inline tests/model_fields/test_uuid.py]: Adding a useful comment for the comments that are only "# regression for #24611" might be out of the scope for this PR but I don't think the change of putting the ticket number followed by a period is an improvement for these cases.

File: tests/admin_changelist/tests.py [hunk 11]
```diff
@@ -111,8 +111,8 @@ def get_list_select_related(self, request):
 
     def test_result_list_empty_changelist_value(self):
         """
-        Regression test for #14982: EMPTY_CHANGELIST_VALUE should be honored
-        for relationship fields
+        EMPTY_CHANGELIST_VALUE should be honored
```
Comment (timgraham) [inline tests/admin_changelist/tests.py]: should be -> is

**Comment by roman-dmytriv** [comment]
Thanks for your comments. I will fix it asap.

**Comment by theodesp** [comment]
Whats the status so far on this one @roman-dmytriv?

**Comment by roman-dmytriv** [comment]
Unfortunately I didn't have enough time to complete the task.
I will have time on the next weekend or at 29-30.
And if you want to complete the task you can reassign it to yourself.

**Comment by theodesp** [comment]
Thank you @roman-dmytriv I really appreciate your help. I will take it as I have spare time this week. Is there anything I would like to know extra or I just review the changes and perform the code review fixes?

**Comment by roman-dmytriv** [comment]
Thank you too @theodesp! Just review the changes and perform the code review fixes.

**Comment by timgraham** [comment]
Closing due to inactivity.

**Pull request closed without merging**